assignments/homework/Raymond Ingram/Homework_3.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[7]:
+
+from IPython import get_ipython
+# get_ipython().run_line_magic('matplotlib', 'inline')
+
+import matplotlib.pyplot as plt
+from astropy import units as u
+from astropy.io import fits
+from astropy.coordinates import SkyCoord
+from astropy.wcs import WCS
+from astropy.visualization import simple_norm
+import matplotlib.pyplot as plt
+
+
+# 
+# <section class="challenge panel panel-success">
+# <div class="panel-heading">
+# <h2><span class="fa fa-pencil"></span> Challenge</h2>
+# </div>
+# 
+# 
+# <div class="panel-body">
+# 
+# <p>Find the position of all the stars in the <a href="data/gaia_lmc_psc.fits">data/gaia_lmc_psc.fits</a> catalog in pixel coordinates, and overplot them on the image (you'll need to use things we have learned in previous tutorials).</p>
+# 
+# </div>
+# 
+# </section>
+# 
+
+# In[2]:
+
+
+plt.rc('image', origin='lower')
+plt.rc('figure', figsize=(10, 6))
+
+# Challenge:
+# Find the position of all the stars in the data/gaia_lmc_psc.fits catalog in pixel coordinates, and overplot them on the image (you'll need to use things we have learned in previous tutorials).
+hduImage = fits.open('data/LMCDensFits1k.fits')
+wcsImage = WCS(hduImage[0].header)
+hduGaia = fits.open('data/gaia_lmc_psc.fits')
+wcsGaia = WCS(hduGaia[1].header)
+ra = hduGaia[1].data['ra']
+dec = hduGaia[1].data['dec']
+starz = SkyCoord(ra*u.deg, dec*u.deg)
+starzOverplot = wcsImage.world_to_pixel(starz)
+ax = plt.subplot(projection=wcsImage)
```

Comment:
This exercise was to work in pixel coordinates, without the WCS projection. 
Drop the projection from here, but that would also mean that you need to drop the lon/lat axes definitions below (which is fine, as you will have x/y pixel coordinates on your axes)